### DISCUSSION TITLE
Add setting to hidden mod to only hide approach circles

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#13109] [Ideas] [answered]
how this mod works is that it works like hidden expect that there is no fading out, so the circles and sliders appear but won't fade out as if it there was no hidden mod applied. however, the approach circles and spinners will act like hidden is on.

the mod will provide a lower score multiplier than the hidden mod

**Comment by peppy** [comment]
This sounds like traceable.

**Comment by huoyaoyuan** [comment]
I'm not sure if it should be considered "easier" than hidden. It should reduce some difficulty of aim, but fading out is a part of reading in hidden.
Should be fair enough as a "fun" mod.

**Comment by Yamakyu** [comment]
Greetings.
Apologizes for bringing this 1 year old issue back, but I was about to open an issue requesting for exactly that and felt it was more appropriate to elaborate here instead.
One thing that OP didn't mention but I think is kinda "implied" in their post is I believe the first circle of a beatmap should be able to display the approach circle, _just like_ in hidden mod indeed.
I mention this as I currently play with a skin that doesn't have approach circles at all (transparent .png), so the 1st circle in silent intros is often a wild guess, timing wise.

I don't really know about it having a score mult at all. For some players (including myself) and in a lot of beatmaps, general gameplay can be thought as easier and more readable without approach circles (less infos on screen). But some beatmaps/patterns are genuinely harder playing that way : for example notes that appear at the exact same position as a previous one clicked earlier (you don't see the clicked circle disappearing, but you don't see the new circle appearing either). Of course this depends on a lot of parameters, and most importantly player skill. Point is I'm not sure removing approach circles makes things specifically harder or easier. 
Though scoring as a whole is a matter beyond my understanding. Just my 2 cents.

That is all, good day, and best of luck with development.

**Comment by bdach** [comment]
@Yamakyu the following setting already exists:

![2021-03-15-200937_526x141_scrot](https://user-images.githubusercontent.com/20418176/111207852-6a67af00-85ca-11eb-8a4b-f513e7e04787.png)

and it makes it so that the approach circle is not faded out for the first object. In my view you've kneecapped your skin in terms of playing hidden by removing them entirely. What does removing them achieve again, exactly?

**Comment by Yamakyu** [comment]
@bdach Ah, I was aware of this setting, but I didn't understand what it does exactly. But now that you mention it, yes I believe it's exactly that.

> What does removing them achieve again, exactly?

Comfort. For me that is. It might be different for OP. But for me it's mostly visual comfort and readability.  
To put it simply, the fact that I can play hidden means I generally have enough timing and spacing information to play without the approach circles (and with fading circles), so they can be removed, and by removing them I make the game easier to read for me most of the times.
For the record I barely play hidden, if at all. But in the rare cases I do, I change the skin for one which has approach circles, yes.

**Comment by Semro** [comment, marked as answer]
https://github.com/ppy/osu/pull/15284
